Description: Reduce the CMake required version to what is in repository
 Upstream has bumped the CMake version required to a newer version than is in
 this distribution, but the version that is available is sufficient for the
 build.
Author: Loren M. Lang <lorenl@north-winds.org>
Origin: other
Forwarded: not-needed
Last-Update: 2024-12-15

--- trustedqsl-2.7.4.orig/CMakeLists.txt
+++ trustedqsl-2.7.4/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 3.14.7 FATAL_ERROR)
+cmake_minimum_required (VERSION 2.8.12.2 FATAL_ERROR)
 
 project(TrustedQSL)
 
